### PR TITLE
@mui v5 compatibility

### LINKED ISSUE #30: @mui v5 compatibility
I'm preparing a Pull Request. The idea would be to release a v2 of `material-ui-snackbar-provider`

### PR DESCRIPTION
**Pull request by ddolcimascolo**
Fixes #30

### CODE REVIEW

**Comment by ddolcimascolo** [comment]
@leMaik Let me know if there's something more you need!

Loking forward for a release including this change :)

**Comment by radokristof** [comment]
When are you going to release this? Can we somehow test this?

**Comment by cgm164** [comment]
Any updates?

**Comment by ddolcimascolo** [comment]
Hey guys, for the record I'm using this exact branch in production with MUI 5 for weeks now without issues. Until this is merged upstream you're pretty much stucked with building/hosting your own version of this module... 

I might assist if you need help

**Comment by KasperAndersson** [comment]
@leMaik Any changes this going to production, any time soon? 🚀

**Comment by laurentlaporte** [comment]
Please merge this!!!

**Comment by laurentlaporte** [comment]
> Hey guys, for the record I'm using this exact branch in production with MUI 5 for weeks now without issues. Until this is merged upstream you're pretty much stucked with building/hosting your own version of this module...
> 
> I might assist if you need help

How did you install this branch in your package json? i tried 

```
"material-ui-snackbar-provider": "TeamWertarbyte/material-ui-snackbar-provider#pull/31",
```

But the `node_modules/material-ui-snackbar-provider` is empty after npm install??

**Comment by arielcarrera** [comment]
Hi guys!, is there any news on this?

**Comment by ddolcimascolo** [comment]
Unfortunately no 😕 Still happy with my fork, though it's sad @leMaik don't merge it...

**Comment by sam-globus** [comment]
Please merge this :+1:

**Comment by leMaik** [comment]
Sorry all, I'll merge it asap! :)

File: test/jestsetup.js
Comment (leMaik) [inline test/jestsetup.js:2]: I guess we should eventually switch to react-testing-library :sweat_smile:

File: package.json
Comment (leMaik) [inline package.json:62]:
```suggestion
    "react": "^17.0.2 || ^18.0.0"
```

**Review by leMaik — APPROVED** [review]

**Comment by leMaik** [comment]
Thank you very much, @ddolcimascolo! :tada:! I'll bump the react peer dependency, maybe update the tests and push out a new major version.

**Comment by leMaik** [comment]
There we go… https://github.com/TeamWertarbyte/material-ui-snackbar-provider/releases/tag/v2.0.0

**Comment by ddolcimascolo** [comment]
Thx for shipping!

**Pull request merged**
